### PR TITLE
ID-0000: Clean repo and fix TemperatureChart

### PR DESCRIPTION
**Pull request by nuria-balaguer**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -96,7 +97,7 @@
     "react": "17.0.1",
     "react-dom": "17.0.1",
     "react-query": "3.17.1",
-    "react-scripts": "4.0.3",
+    "react-scripts": "4.0.1",
```
Comment (dimaPortyanka) [inline package.json:100]: why react scripts downgrade?

File: src/stories/FailureMessage.stories.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+import FailureMessage from 'TemperatureChart2/FailureMessage'
+
+const FailureMessageStory = {
+    title: 'FailureMessage',
+    component: FailureMessage,
+    argTypes: {
+        img: {
+            control: 'string',
+        },
```
Comment (dimaPortyanka) [inline src/stories/FailureMessage.stories.js:11]: this is useless

File: package.json (hunk 1 above)
Comment (nuria-balaguer) [reply package.json:100]: I couldn't make it work with the 4.0.3, it said this: 
Screenshot 2021-08-05 at 12 52 26

File: src/stories/FailureMessage.stories.js (hunk 2 above)
Comment (nuria-balaguer) [reply src/stories/FailureMessage.stories.js]: fixed

**Review by dimaPortyanka — APPROVED** [review]

**Pull request merged**